platforms/__common.sh

```diff
@@ -18,7 +18,13 @@ blobExists() {
 	local blobName="$2"
 	local sdkStorageAccountUrl="$3"
 	local exitCode=1
-	curl -I $sdkStorageAccountUrl/$containerName/$blobName 2> /tmp/curlError.txt 1> /tmp/curlOut.txt
+	sasToken=""
+	if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
+		set +x
+		sasToken=$ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN
+		set -x
+	fi
+	curl -I $sdkStorageAccountUrl/$containerName/$blobName$sasToken 2> /tmp/curlError.txt 1> /tmp/curlOut.txt
```

Comment:
@daniv-msft this is the change I made to fix the `PlaformBinaries` Pipeline. 

The sas-token was not passed in the `http` request and the request was failing and this particular request is to check if any sdk already exists in the `staging` storage account. As the request was failing, the pipeline was uploading all the sdks to the `staging` account. That's why it was taking hours (nearly a day) to complete the pipeline. 
After this fix, the builds in the pipeline completed in just 20 minutes.